__tests__/api-endpoints.test.js

```diff
@@ -190,3 +190,117 @@ describe("/api/articles/:article_id/comments - sad path", () => {
   })
 });
 
+describe("POST - /api/articles/:article_id/comments", () => {
+  test("responds with the posted comment, an object with username & body key / values", () => {
```

Comment:
I would maybe add in this test description that the server will respond with a 201 and in what circumstance it will respond with the object eg. if given a valid comment